config/schema/field_definitions.json

```diff
@@ -345,6 +345,38 @@
     "type": "float"
   },
 
+  "fault_type": {
+    "type": "identifiers"
+  },
+
+  "item_type": {
```

Comment:
Thanks for pointing this out. It is not always a vehicle. I could rename to `recalled_item_*`, `faulty_item_*`, any other suggestions?